pkg/nodes/node.go

```diff
@@ -189,6 +195,49 @@ func (builder *NodeBuilder) ExternalIPv4Network() (string, error) {
 	return extNetwork.IPv4, nil
 }
 
+// IsReady check if the Node is Ready.
+func (builder *NodeBuilder) IsReady() bool {
+	if valid, _ := builder.validate(); !valid {
+		return false
+	}
+
+	glog.V(100).Infof("Verify %s node availability", builder.Definition.Name)
+
+	var err error
+	builder.Object, err = builder.Pull()
```

Comment:
Please re-place this method by func Exists. It'll update builder.Object automatically

Reply:
done